test/commands/extended/utils_test.py

```diff
@@ -239,3 +245,5 @@ def test_multiple_addresses_return(self):
                 },
             ]
         )
+
+# TODO: add tests for `get_bundles_from_transaction_hashes`
```

Comment:
❔ Should we capture this with a GitHub issue, so that we don't forget about it?

Reply:
yep, see #302

Reply:
Oh, right 😅